spec/components/pair_request/status_buttons_component_spec.rb

```diff
@@ -3,35 +3,64 @@
 require 'rails_helper'
 
 RSpec.describe PairRequest::StatusButtonsComponent, type: :component do
-  let!(:current_user) { create(:user) }
-  let!(:pending_request) { create(:pair_request, invitee: current_user) }
-  let!(:completed_request) { create(:pair_request, invitee: current_user, status: :completed) }
-  let!(:request_awaiting_completion) do
-    create(:pair_request, author: current_user, when: 5.minutes.ago, status: :accepted)
-  end
+  let(:invitee) { create(:user) }
+  let(:author) { create(:user) }
+
+  context 'when the current user is the invitee' do
+    let(:current_user) { invitee }
 
-  context 'with a pair request that can be accepted or rejected' do
-    it 'renders an Accept button and a Reject button' do
-      render_inline(described_class.new(pair_request: pending_request, current_user:))
+    context 'with a pair request that can be accepted or rejected' do
+      it 'renders an Accept button and a Reject button' do
+        pending_request = create(:pair_request, invitee: current_user)
```

Comment:
vars locally scoped here on purpose, with an eye towards the rationale provided in my lint suggestions: https://github.com/agency-of-learning/PairApp/pull/108#issuecomment-1599679720